macos/FluentUITestViewControllers/TestViewControllers.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+public struct TestViewController: Identifiable {
+	public let id: String
```

Comment:
title?

Reply:
id is needed to conform to Identifiable, which is needed to be in a ForEach struct in SwiftUI. I was kinda being lazy by just purposing the name thing to id, but the more technically correct thing (maybe a more more clunky but more obvious) is probably to have a separate id property that we enforce as unique. Will do that in the next commit. Let me know what you think.